include/seqan3/alphabet/aminoacid/aa10murphy.hpp

```diff
@@ -0,0 +1,238 @@
+            for (auto & c : ret)
+                c = 9; // value of 'S', because that appears most frequently
+
+            // reverse mapping for characters and their lowercase
+            for (rank_type rnk = 0u; rnk < value_size; ++rnk)
+            {
+                ret[static_cast<rank_type>(         rank_to_char[rnk]) ] = rnk;
+                ret[static_cast<rank_type>(to_lower(rank_to_char[rnk]))] = rnk;
+            }
+
+            ret['D'] = ret['B']; ret['d'] = ret['B']; // Convert D to B (either D/N).
+            ret['E'] = ret['B']; ret['e'] = ret['B']; // Convert E to B (either D/N).
+            ret['J'] = ret['I']; ret['j'] = ret['I']; // Convert J (either I/L) to I.
+            ret['L'] = ret['I']; ret['l'] = ret['I']; // Convert L to I.
+            ret['M'] = ret['I']; ret['m'] = ret['I']; // Convert M to I.
+            ret['N'] = ret['B']; ret['n'] = ret['B']; // Convert N to B (either D/N).
+            ret['O'] = ret['K']; ret['o'] = ret['K']; // Convert Pyrrolysine to A (either D/N).
+            ret['Q'] = ret['B']; ret['q'] = ret['B']; // Convert Q to B (either D/N).
+            ret['R'] = ret['K']; ret['r'] = ret['K']; // Convert R to K (either D/N).
+            ret['T'] = ret['S']; ret['t'] = ret['S']; // Convert T to S (either D/N).
+            ret['U'] = ret['C']; ret['u'] = ret['C']; // Convert Selenocysteine to B (either D/N).
+            ret['V'] = ret['I']; ret['v'] = ret['I']; // Convert D to I (either D/N).
+            ret['W'] = ret['F']; ret['w'] = ret['F']; // Convert W to F (either D/N).
+            ret['X'] = ret['A']; ret['x'] = ret['A']; // Convert unknown amino acids to Serine.
+            ret['Y'] = ret['F']; ret['y'] = ret['F']; // Convert Y to E.
+            ret['Z'] = ret['B']; ret['z'] = ret['B']; // Convert Z (either E/Q) to B (either D/N).
+            ret['*'] = ret['F']; // The most common stop codon is UGA. This is most similar to a Tryptophan which in this alphabet gets converted to Phenylalanine.
+            return ret;
+        }()
+    };
+};
+
+} // namespace seqan3
+
+// ------------------------------------------------------------------
+// metafunctions
+// ------------------------------------------------------------------
+
+namespace seqan3
+{
+
+//!\brief Helper metafunction that identifies aa10murphy as an amino acid alphabet.
+//!\ingroup aminoacid
+template <>
+struct is_aminoacid<aa10murphy> : std::true_type {};
+
+} // namespace seqan3
+
+// ------------------------------------------------------------------
+// containers
+// ------------------------------------------------------------------
+
+namespace seqan3
+{
+//!\brief Alias for an std::vector of seqan3::aa10murphy.
+//!\relates aa10murphy
+using aa10murphy_vector = std::vector<aa10murphy>;
+
+} // namespace seqan3
+
+// ------------------------------------------------------------------
+// literals
+// ------------------------------------------------------------------
+
+namespace seqan3
+{
```

Comment:
you are opening and closing namespace seqan3 really often here. You can just open it once at the top of the file and close it once at the bottom